t/plugin/log-rotate2.t

```diff
@@ -181,13 +181,13 @@ plugins:
   - log-rotate
 plugin_attr:
   log-rotate:
-    interval: 1
+    interval: 2
     max_kept: 1
     enable_compression: true
 --- config
     location /t {
         content_by_lua_block {
-            ngx.sleep(3.5)
+            ngx.sleep(2.5)
```

Comment:
why is this change needed?

Reply:
The tests are flaky and fail intermittently and with this timeout configuration they stopped failing.